src/lib/get-token.js

```diff
@@ -46,12 +34,11 @@ async function getToken(store, client, auth, state) {
       sub: auth.userId,
       aud: client.clientId
     };
-    if (email) {
-      payload.email = email;
-    }
-    if (phone) {
-      payload.phone = phone;
-    }
+    identifiers.forEach(identifier => {
+      const splitted = identifier.split(':');
+      // eslint-disable-next-line prefer-destructuring
+      payload[splitted[0]] = splitted[1];
```

Comment:
How about this:

```suggestion
      const [ key, value ] = identifier.split(':');
      payload[key] = value;
```

... that's what array destructuring is made for ;)